README.md

```diff
@@ -1,6 +1,55 @@
+<div align="center">
+
+<a href=https://www.mia-platform.eu/>
+<img alt="logo" src="https://raw.githubusercontent.com/mia-platform/.github/master/profile/img/mia-platform_logo_color.png" height="200">
+</a>
+
 # Console SDK
 
-This repository holds all the packages for the Mia-Platform Console SDK useful to create custom extensions to further improve your Console experience!
+[![NPM publish CI][action-status-svg]][github-action]
+[![javascript style guide][standard-mia-svg]][standard-mia]  
+[![Coverage Status][coverall-svg]][coverall-io]
+
+</div>
+
+
+This repository is used to configure your microfrontend, as Mia platform Console extensions, easily
+See the specific monorepo readme to using this library
+
+## Monorepos
+
+### Console SDK. 
+An abstract interface used within helper repositories. 
+[Read more](./packages/microfrontend-sdk)                     
+
+### Vite helpers.
+Used to configure the microfrontend with vite. 
+[Read more](./packages/vite-helpers-console-microfrontend)    
+
+### Webpack helpers. 
+Used to configure the microfrontend with webpack. 
+[Read more](./packages/webpack-helpers-console-microfrontend)
+
+## Install
+
+To use this repository 
+
+```bash
+npm install @mia-platform-internal/console-sdk
```

Comment:
Is it true? We won't have different packages

Reply:
after changeset tag we have differents tags to use all of this packages

Reply:
i update the docs to correctly indicates how to install repository